docs/content/release-notes/flink-1.15.md

```diff
@@ -0,0 +1,379 @@
+the behavior of 1.13 to be consistent with Hive/Spark.
+
+### [Use the new casting rules in TableResult#print (FLINK-24685)](https://issues.apache.org/jira/browse/FLINK-24685)
+
+The string representation of `BOOLEAN` columns from DDL results 
+(`true/false -> TRUE/FALSE`), and row columns in DQL results 
+(`+I[...] -> (...)`) has changed for printing.
+
+### [Sinks built with the unified sink framework do not receive timestamps when used in Table API (FLINK-24608)](https://issues.apache.org/jira/browse/FLINK-Table)
+
+This adds an additional operator to the topology if the new sink interfaces are used 
+(e.g. for Kafka). It could cause issues in 1.14.1 when restoring from a 1.14 savepoint. 
+A workaround is to cast the time attribute to a regular timestamp in the SQL statement
+closely before the sink.
+
+### [SQL functions should return `STRING` instead of `VARCHAR(2000)` (FLINK-24586)](https://issues.apache.org/jira/browse/FLINK-24586)
+
+Functions that returned `VARCHAR(2000)` in 1.14, return `VARCHAR` with maximum 
+length now. In particular this includes:
+
+```SQL
+SON_VALUE
+CHR
+REVERSE
+SPLIT_INDEX
+REGEXP_EXTRACT
+PARSE_URL
+FROM_UNIXTIME
+DECODE
+DATE_FORMAT
+CONVERT_TZ
+```
+
+## Connectors
+
+### [(Introducing GCS FileSystem Support and) Update GCS documentation (FLINK-25577)](https://issues.apache.org/jira/browse/FLINK-25577)
+
+The `flink-gs-fs-hadoop` FileSystem plugin for Google Cloud Storage (GCS) has been
+introduced. This allows Flink to read data from and write data to GCS via paths with 
+the `gs://` scheme, and it provides similar functionality for GCS as, for example, 
+the `flink-s3-fs-hadoop` provides for Amazon S3.
+
+In particular, this plugin supports the `RecoverableWriter` interface, which 
+allows it to be used with file sinks.
+
+Under the hood, the `flink-gs-fs-hadoop` uses Google's `gcs-connector` Hadoop 
+library for basic read/write operations, and it uses Google's `google-cloud-storage` 
+library to implement `RecoverableWriter` functionality.
+
+### [Remove MapR filesystem (FLINK-25553)](https://issues.apache.org/jira/browse/FLINK-25553)
+
+Support for the MapR FileSystem has been dropped.
+
+### [Make flink-connector-base dependency usage consistent across all connectors (FLINK-25927)](https://issues.apache.org/jira/browse/FLINK-25927)
+
+`flink-connector-base` was previously inconsistently used in connectors (directly 
+shaded in some and transitively pulled in via `flink-connector-files` which was 
+itself shaded in the table uber jar). 
+[FLINK-24687](https://issues.apache.org/jira/browse/FLINK-24687) moved 
+`flink-connector-files` 
+out of the `flink-table` uber jar. In order to address this, Flink 1.15 implements
+a combined approach for both making usage of `flink-connector-base` consistent 
+and ensuring a smooth transition for both Flink users and for external connector
+developers:
+
+1. all internal Flink connectors that depend on `flink-connector-base` now shade
```

Comment:
this was actually partially reverted; cc @AHeise @afedulov

Reply:
Yes, the commit with the associated release note was reverted.

Reply:
Some background: unfortunately some classes within the `flink-connector-base`,  such as `DeliveryGuarantee`, that are public and can be used in the user code and need to be first moved into `flink-core`. The same applies to the `HybridSource`.

Reply:
Thanks @afedulov for the information! I'll then first remove this issue from the release note.